pkg/strategy/controller.go

```diff
@@ -183,7 +183,7 @@ func normalizeElasticsearch(spec *v1.ElasticsearchSpec) {
 func normalizeRollover(spec *v1.JaegerEsRolloverSpec) {
 	spec.Image = util.ImageName(spec.Image, "jaeger-es-rollover-image")
 	if spec.Schedule == "" {
-		spec.Schedule = "*/30 * * * *"
+		spec.Schedule = "0 3 * * *"
```

Comment:
Any reason not to use `@daily`? It's typically better to use this kind of notation, as it allows the scheduler to spread jobs across the day, to avoid all jobs to run at the same time.

Reply:
I wanted to pick the lowest traffic time rather than run it at random time.

Reply:
Do you mean that Kubernetes will run the same job at random times when `@daily` is set? That's certainly not in line with other cron implementations...

Reply:
Well didn't you just say it? 

>  as it allows the scheduler to spread jobs across the day, to avoid all jobs to run at the same time.

I don't know how it is implemented. But I think it's better to run the rollover in low traffic hours.

Reply:
No, what I said is that the scheduler can spread the *jobs* (not executions) across the day. This way, your job might run every day at 4:07am (give or take).

Reply:
> low traffic hours.

3am for some might be prime time for others :-)

Reply:
I see thanks for the explanation. Once the time set it will stick with that.

Reply:
As these are default values, I would recommend setting to midnight, as this is a natural day boundary. Then if users want something else, they will need to specify.